data/hlint.yaml

```diff
@@ -1108,6 +1113,7 @@
 # no  = x `elem` y
 # no  = elem 1 [] : []
 # yes = a & (mapped . b) %~ c -- a <&> b %~ c
+# yes = Language.Haskell.TH.varE 'foo -- [|foo|]
```

Comment:
Final nit - can we get a non-qualified example too, since otherwise we run the risk of missing real users code

Reply:
Sure thing, made the test use it non-qualified